service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -134,4 +137,6 @@ private final Respondent getFirstRespondent() {
     private final JudgeAndLegalAdvisor judgeAndLegalAdvisor;
     private final C2DocumentBundle temporaryC2Document;
     private final List<Element<C2DocumentBundle>> c2DocumentBundle;
+    private final C21Order temporaryC21Order;
```

Comment:
hmm is this naming right? Sure it's temporary in the UI, but here, it's a C21Order, no?

Reply:
true I got a bit confused with the naming. I think we could call it c21Order here and that would be okay. what about the order bundle? better name for that?

Reply:
I still think you could do this with one object, a c21Order which is an individual order, and then List<Element<c21Order>> which is your collection you display

Reply:
I agree - but then what about e.g orderDetails, which I don't want to display in the tab, but I do want to display in the screens + check answers. is there a way to hide this only in the tab? also with my own custom bundle object for the tab only it means I can format it however I want (e.g can combine judge title and name into one line, can name the field labels differently from how they are in the screens, etc). If I can hide the details in the tab (but only the tab), then yes this sounds good